src/css-length-value.js

```diff
@@ -123,5 +122,8 @@
     throw new TypeError('Should not be reached');
   };
 
+  internal.CSSLengthTypes = CSSLengthTypes;
+  internal.isValidLengthType = isValidLengthType;
+
   scope.CSSLengthValue = CSSLengthValue;
```

Comment:
This is not introduced in your change, but I just noticed it. The spec doesn't expose a CSSLengthValue constructor, only `from`:

https://drafts.css-houdini.org/css-typed-om-1/#lengthvalue-objects